etcdserver/api/v3rpc/grpc.go

```diff
@@ -36,7 +36,10 @@ func Server(s *etcdserver.EtcdServer, tls *tls.Config) *grpc.Server {
 	}
```

Comment:
this patch should be labeled `v3rpc: ...` instead of `etcdserver: ...` since it touches the v3rpc package